src/shared/sam/cli/samCliLocator.ts

```diff
@@ -136,7 +136,11 @@ class WindowsSamCliLocator extends BaseSamCliLocator {
 }
 
 class UnixSamCliLocator extends BaseSamCliLocator {
-    private static readonly LOCATION_PATHS: string[] = ['/usr/local/bin', '/usr/bin']
+    private static readonly LOCATION_PATHS: string[] = [
+        '/usr/local/bin',
+        '/usr/bin',
+        `${process.env.HOME}/.linuxbrew/bin`,
```

Comment:
This path is incorrect. Linuxbrew installs SAM under `/home/linuxbrew/.linuxbrew/bin/sam`, regardless of the user's home directory.

Reply:
To be precise, "The installation script installs Homebrew to /home/linuxbrew/.linuxbrew using sudo if possible and in your home directory at ~/.linuxbrew otherwise." (https://docs.brew.sh/Homebrew-on-Linux)

So I think both `${process.env.HOME}/.linuxbrew/bin` and `/home/linuxbrew/.linuxbrew/bin` should be included.

Reply:
Thank you for the references, PR updated. Woe the user that actually chooses "linuxbrew" as their username...